src/index.js

```diff
@@ -80,6 +80,7 @@ function getParent (element) {
   }
   // if an element is inside the shadow DOM, break outside of it
   const rootNode = element.getRootNode()
+  /* istanbul ignore else */
```

Comment:
Coverage fixes are orthogonal to `getElementById` and should probably go in another PR.

Reply:
@nolanlawson I removed the coverage fixes from this PR